dist/BUILD.bazel

```diff
@@ -0,0 +1,121 @@
+load("@build_bazel_rules_nodejs//:defs.bzl", "rollup_bundle")
+load("//lib:bundle.bzl", "ES2015_BUNDLES", "ES5_BUNDLES", "ES5_GLOBAL_BUNDLES")
+
+package(default_visibility = ["//:__subpackages__"])
+
+# copy this file from //lib to //dist
+genrule(
+    name = "zone_externs",
+    srcs = ["//lib:closure/zone_externs.js"],
+    outs = ["zone_externs.js"],
+    cmd = "cp $< $@",
+)
+
+genrule(
+    name = "zone_d_ts",
+    srcs = ["//lib"],
```

Comment:
is this concatenating multiple .d.ts inputs into a single output? maybe we should use the api-extractor like we do in the angular repo.

Reply:
This `rule` just copy the `zone.d.ts` from `lib` to `dist/zone.js.d.ts`, we only have one `d.ts` file here.